src/app.js

```diff
@@ -113,6 +115,28 @@ class Application {
       }
     );
 
+    expressApp.post(
+      '/convert/html',
+      check('collection')
+        .exists()
+        .withMessage(
+          'The JSON summary must have postman collection information.'
+        ),
+      check('run')
+        .exists()
+        .withMessage('The JSON summary must have collection run data.'),
+      check('run.executions')
+        .exists()
+        .withMessage('The collection run data is not valid.'),
+      (req, res) => {
+        if (!this.validateInput(req, res)) return;
+
+        var htmlReport = generateHTMLReport(req.body);
+        res.set('Content-Type', 'text/html');
+        res.send(Buffer.from(htmlReport));
```

Comment:
wrap with a try/catch, to prevent the whole server dying if anything wrong happen

Reply:
the content of generateHTMLReport() is already wrapped in try/catch.
Will that not be enough?

Reply:
I'm worried about the res.send(), as I don't know what would happen if a client timeout/close the connection.

Reply:
Update: I removed the block from the function and put it where you asked.